content/blog/2023-12-20-changepoint_explore.Rmd

```diff
@@ -0,0 +1,91 @@
+
+<center>
+![**Figure 1.** Total and Variant Case Counts with variant changepoints on April 24, 2021; June 19, 2021; December 18, 2021; March 5, 2022; June 18, 2022.](/blog/2023-12-20-changepoint_explore_files/image3.png)
+</center>
+
+## Methods
+
+Next, we needed to identify how these ground truth changepoints compared to changepoints in indicators (using the same three algorithms). We used all available data streams during our selected time range from Delphi’s [COVIDCast database](https://cmu-delphi.github.io/delphi-epidata/), which corresponded to 60 indicators (including the Facebook survey, Google Search, claims, hospitalization, and mortality indicators) each at the state, territory, and national level. When we aggregated the calculated changepoints per indicator per week, as Fig. 2 shows, we identified that some weeks have many more changepoints than others. To match the number of ground truth changepoints, we filter to the top 5 weeks per indicator (major indicator changepoints) to directly compare with the ground truth changepoints. 
+
+<center>
+![**Figure 2.** The aggregated count of changepoints per week in all fifty two regions by all three algorithms for estimated COVID related outpatient doctor visits](/blog/2023-12-20-changepoint_explore_files/image5.png)</center>
+
+Then, we categorized indicators as early, on-time, or late using the following definitions: 
+
+- **Early**: The five major indicator changepoints consistently precede the ground truth changepoints. 
+- **On-time**: The major indicator changepoints appear within a week of the ground truth changepoints
+- **Late**: The  major indicator changepoints lag behind the ground truth changepoints by more than a week. 
+
+As we expected each wave of the pandemic to last around four weeks, we necessitate that any major indicator changepoints found with a distance greater than 28 days to the nearest ground truth changepoint have no relationship with the ground truth changepoint.
+
+## Results and Analysis
+
+In total, of the sixty indicators, 4 were early, 8 were on time, and 12 were late. There was also 1 undetermined indicator and 35 indicators whose major indicator changepoints had inconsistent relationships with the ground truth changepoints. Some indicators in this inconsistent category started out as early indicators and ended as late indicators, while others started as late indicators but ended as on time indicators, showing that indicators may not always be consistently early, on time, or late. 
+Interestingly, indicators from the same source were usually classified similarly, suggesting that the indicator timeliness depends heavily on the data provider that serves as the source for the indicators. Still, while there was some variance in the leadingness of early indicators, other categories had much less variance. 
```

Comment:
the first sentence is longer than it needs to be.  also maybe put "leadingness" in quotes since its not a real word?  alternatively, you could say "leading nature" instead of "leadingness" (without quotes).
```suggestion
Interestingly, indicators from the same source were usually classified similarly, suggesting that indicator timeliness depends heavily on the data provider. Still, while there was some variance in the "leadingness" of early indicators, other categories had much less variance. 
```

Reply:
This reads a lot more smoothly - thank you!